src/navigations/Flow3Navigator/index.js

```diff
@@ -1,18 +1,32 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { createStackNavigator } from '@react-navigation/stack'
 import { useNavigation } from '@react-navigation/native'
 import Flow3 from '../../screens/BenefitListFlow3/Flow3'
+import ReviewBenefitList from '../../screens/BenefitListFlow3/ReviewBenefitList'
 import ReflectionHome from '../../screens/ReflectionHome'
 import Header from '../../components/Header'
 import BenefitProgressHeader from '../../components/BenefitProgressHeader'
 import PromptAudio from '../../screens/SucHab01Screens/PromptAudio'
 import PromptVideo from '../../screens/SucHab01Screens/PromptVideo'
 import PromptText from '../../screens/SucHab01Screens/PromptText'
+import PostActivity from '../../screens/SucHab01Screens/PostActivity'
+import BreatheScreen from '../../screens/SucHab01Screens/Breathe'
+import PreviewScreen from '../../screens/SucHab01Screens/PreviewScreen'
+import VideoPlayer from '../../screens/PositiveThinking/video'
 import { content } from '../../../assets/content'
 
 const Stack = createStackNavigator()
+const { prompts, postActivity } = content.successfulHabits

```

Comment:
You are bringing in content for Successful Habits. You have to hard code your prompts or create an additional content object. The prompts going to TextPrompt and VideoPrompt are for a different exercise

Reply:
I hardcoded the prompts that were different from successful habit but I can create separate content for it